stress/common/src/main/java/alluxio/stress/fuse/FuseIOSummary.java

```diff
@@ -50,25 +42,21 @@ public FuseIOSummary() {
    * @param parameters the parameters for the Fuse IO stress bench
    * @param baseParameters the base parameters for the Fuse IO stress bench
    * @param nodes the unique ids of all job workers
-   * @param errors all errors reported by job workers
    * @param recordStartMs the timestamp starting counting bytes
    * @param endMs the timestamp that the test ends
    * @param ioBytes total number of bytes processed by workers
    * @param ioMBps aggregated throughput data
-   * @param clientsThroughput the throughput data of each job worker
    */
   public FuseIOSummary(FuseIOParameters parameters, BaseParameters baseParameters,
-      List<String> nodes, Map<String, List<String>> errors, long recordStartMs, long endMs,
-      long ioBytes, float ioMBps, Map<String, Float> clientsThroughput) {
-    mClientsThroughput = clientsThroughput;
-    mNodes = nodes;
-    mErrors = errors;
+      Map<String, FuseIOTaskResult> nodes, long recordStartMs, long endMs,
+      long ioBytes, float ioMBps) {
+    mNodeResults = nodes;
     mParameters = parameters;
     mBaseParameters = baseParameters;
     mRecordStartMs = recordStartMs;
     mEndMs = endMs;
     mIOBytes = ioBytes;
-    mIOMBps = ioMBps;
+    mThroughput = ioMBps;
```

Comment:
I'm actually not sure if this `mIOMbps` can be replaced by using `mThroughput` because they might have different definitions. For compatibility, can you set `mThroughput = mIOMbps = ioMBps`? Let's keep this field for now.

@ssz1997 @LuQQiu FYI

Reply:
OK. In addition to FuseIOSummary, I also restore the getter and setter for mIOMbps in WorkerBenchSummary in the same reason.